python/ray/autoscaler/local/development-example.yaml

```diff
@@ -1,11 +1,7 @@
 cluster_name: default
-min_workers: 0
-max_workers: 0
```

Comment:
Don't need to demonstrate non-default autoscaling settings anywhere except example-full.